definitions/npm/antd_v3.x.x/flow_v0.25.x-v0.103.x/antd_v3.x.x.js

```diff
@@ -601,6 +601,19 @@ declare module "antd" {
 
   declare export class Tooltip extends React$Component<TooltipProps> {}
 
+  declare export class Tree extends React$Component<TreeProps, any> {
```

Comment:
`TreeProps` is not defined, you'll need to declare that type as well.   
Maybe just copy/paste it from their TS types.  

Also remove the `any` here, the state generic param is not mandatory.